grpc/py-cubic/cubic/test_client.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright(2019) Cobalt Speech and Language Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http: // www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import io
+import grpc
+from concurrent import futures
+
+from client import Client, RecognitionAudio, RecognitionConfig
+import cubic_pb2
+import cubic_pb2_grpc
+
+expectedResponses = {}
+expectedResponses['Version'] = cubic_pb2.VersionResponse(cubic='2727', server='v.27.0')
+expectedResponses['ListModels'] = cubic_pb2.ListModelsResponse(models=[cubic_pb2.Model(id="1"), cubic_pb2.Model(id="2")])
+expectedResponses['Recognize'] = cubic_pb2.RecognitionResponse(results=[cubic_pb2.RecognitionResult(alternatives=[cubic_pb2.RecognitionAlternative(transcript="This is a test")])])
```

Comment:
This line feels a little long.  Is there an easy way to to multi-line definitions?  I know I had a hard time in go deciding on the best format, so I understand if you say this is the best way for python

Reply:
true, it is long, I did break it into multiple lines but it looked ugly in my eyes. I will try and see if I can make it look cleaner